pkg/credentialstatus/credentialstatus.go

```diff
@@ -54,3 +63,27 @@ func (v *Verifier) Verify(vc *verifiable.Credential) error {
 
 	return nil
 }
+
+type wrapResolver struct {
+	resolver api.DIDResolver
+}
+
+func (w *wrapResolver) Resolve(did string, _ ...vdr.DIDMethodOption) (*diddoc.DocResolution, error) {
+	return w.resolver.Resolve(did)
+}
+
+func (w *wrapResolver) Create(string, *diddoc.Doc, ...vdr.DIDMethodOption) (*diddoc.DocResolution, error) {
+	return nil, fmt.Errorf("create operation not supported")
+}
+
+func (w *wrapResolver) Update(*diddoc.Doc, ...vdr.DIDMethodOption) error {
+	return fmt.Errorf("update operation not supported")
+}
+
+func (w *wrapResolver) Deactivate(string, ...vdr.DIDMethodOption) error {
+	return fmt.Errorf("deactivate operation not supported")
```

Comment:
Just wondering if omitting the verb `is` is a common practice

Reply:
idk
added it